src/Settings.cs

```diff
@@ -11,15 +11,15 @@ namespace Dictionary
     public class Settings
     {
         public string ConfigFile;
-        public string ICIBAToken = "BEBC0A981CB63ED5198597D732BD8956";
-        public string BighugelabsToken = "";
-        public int MaxEditDistance = 3;
-        public bool ShowEnglishDefinition = false;
-        public string WordWebsite = "";
+        public string ICIBAToken { get; set; } = "BEBC0A981CB63ED5198597D732BD8956";
+        public string BighugelabsToken { get; set; } = "";
+        public int MaxEditDistance { get; set; } = 3;
+        public bool ShowEnglishDefinition { get; set; } = false;
+        public string WordWebsite { get; set; } = "";
 
-        public async void Save()
+        public void Save()
         {
-            await JsonSerializer.SerializeAsync(File.OpenWrite(ConfigFile), this);
+            File.WriteAllText(ConfigFile, JsonSerializer.Serialize(this));
```

Comment:
Changed it back to blocking api because there seems to be some kind of race condition otherwise.

Reply:
I think this logic should be changed later after `ISavable` has been migrated to `Flow.Launcher.Plugin` so that we shall allow the call of Save automatically by Flow.

Reply:
That sounds handy! Look forward to that and `PluginJsonStorage` class you mentioned.

Reply:
@harrynull 
It is on progress.
https://github.com/Flow-Launcher/Flow.Launcher/pull/341